### DISCUSSION TITLE
Co-ordinate displays not showing.

### DISCUSSION THREAD

**Discussion by Quartyz** [winder/Universal-G-Code-Sender#1923] [Help]
Recently had to re-install the computer (hardware problem) - Linux Mint 20.3. I use UGS a lot, to feed an XYZ router, and until the crash, had been running UGS 2.0.7 Platform no problem - I like the 'If it ain't broke, don't fix it' theory.
Re-installation was with 2.0.11 Platform. When I tried to use it, I noticed the XYZ C-ordinate displays became blank while UGS was in operation. When the process was finished, they showed the appropriate values. When I tried to jog the machine, the displays disappeared during the jogging movement, and re-appeared on completion, with the correct reading.
I suspected maybe a faulty installation, so entirely removed all traces of UGS I could find, re-downloaded, and re-installed - but still the same problem.
A second machine, running Linux Mint 20.1, shows the same problem, and also a laptop using Linux Mint modified to operate with the latest Linux Kernel (Liquorix).
Finally, I tried installing first UGS 20.0.9, but this still had the co-ordinates display problem. Desperation! - so I tried UGS 2.0.7 (Platform), and it works fine - the co-ordinates show while the machine is running, and display correctly.
So - any suggestions, or is it a bug?

**Comment by breiler** [comment]
This can very well be a bug affecting certain variants of Linux, I wasn't able to reproduce it on Ubuntu 22.04.
I need to set up a virtual box to test this, so that might take some time to investigate.

**Comment by Quartyz** [comment]
Thanks for your quick reply - I wasn't certain it was a bug - I'm happy
enough with V2.0.7 for now. If I can help with any info - no problems.

All the best

Kith
